.travis.yml

```diff
@@ -231,7 +230,10 @@ before_install:
         conda config --prepend channels conda-forge --prepend channels symengine/label/dev;
 
         conda info -a;
-        conda create -q -n test-environment python=$TRAVIS_PYTHON_VERSION pip ${TEST_OPT_DEPENDENCY/sage};
+        CONDA_PKGS=${TEST_OPT_DEPENDENCY/sage}
+        CONDA_PKGS=${CONDA_PKGS/autowrap/libgfortran libgcc gcc cython}
+        conda create -q -n test-environment python=$TRAVIS_PYTHON_VERSION pip ${CONDA_PKGS};
+        pip install wurlitzer;  # package for capturing C level output (tests for stdin/stderr)
```

Comment:
What is wurlitzer? Can we get a conda-forge package for it?

Reply:
It's for testing compiled binaries of generated C code. Here's the package:
https://github.com/minrk/wurlitzer
I opened a PR against staged-recipes:
https://github.com/conda-forge/staged-recipes/pull/6195